tools/java/.settings/org.eclipse.wst.common.component

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?><project-modules id="moduleCoreId" project-version="1.5.0">
```

Comment:
What is this about: project-version="1.5.0"?

Reply:
It's managed by Eclipse. This is automatically generated file, so I think we don't need to worry about version much.

Reply:
> Looks good to me.
> 
> One comment - do we want to use "java" as component folder name? I prefer to use folder name matching artifact ID.

Yes, I'd like to rename it, maybe `cldr-utils`? Not sure.